bears/general/InvalidLinkBear.py

```diff
@@ -10,22 +10,23 @@
 class InvalidLinkBear(LocalBear):
 
     @classmethod
-    def check_prerequisites(cls):
-        code = InvalidLinkBear.get_status_code_or_error("http://www.google.com")
+    def check_prerequisites(cls, timeout=2):
```

Comment:
no please remove it, and supply `2` directly below for the timeout. Or even better, make a `DEFAULT_TIMEOUT = 2` field and use that for the default inside `run()` and here :+1: